bundles/org.openhab.binding.bluetooth.airthings/src/main/resources/OH-INF/thing/airthings.xml

```diff
@@ -69,6 +69,30 @@
 			</parameter>
 		</config-description>
 	</thing-type>
+	<thing-type id="airthings_wave_gen1">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="roaming" />
+			<bridge-type-ref id="bluegiga" />
+			<bridge-type-ref id="bluez" />
+		</supported-bridge-type-refs><label>Airthings Wave Gen 1</label><description>Smart Radon Monitor</description><channels>
```

Comment:
Some line breaks seem to be missing here.
Please run a `mvn spotless:apply`.

Reply:
I made the corrections.